src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/misc/ModuleAutoInsult.kt

```diff
@@ -0,0 +1,96 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.ccbluex.liquidbounce.features.module.modules.misc
+
+import net.ccbluex.liquidbounce.event.events.ChatReceiveEvent
+import net.ccbluex.liquidbounce.event.handler
+import net.ccbluex.liquidbounce.features.module.Category
+import net.ccbluex.liquidbounce.features.module.Module
+import net.ccbluex.liquidbounce.config.Choice
+import net.ccbluex.liquidbounce.config.ChoiceConfigurable
+
+
+/**
+ * Auto insult on kill module
+ *
+ * Sends an insult in chat when someone is killed by you.
+ */
+object ModuleAutoInsult : Module("AutoInsult", Category.MISC) {
+
+
+    private val modes = choices(
+        "Mode", Toxic, arrayOf(
+            Toxic, LiquidBounce
+        )
+    )
```

Comment:
Please use `enumChoice` instead and put the insult list in an array in an enum field. It will make the code look much simpler